pcap/pcap.h

```diff
@@ -37,6 +37,10 @@
 
 #include <pcap/export-defs.h>
 
+#if defined(WIN32) && _MSC_VER < 1900
```

Comment:
> How do you suggest to solve it then?

Define `HAVE_SNPRINTF` only if you actually have `snprintf()`.

Maybe it's impossible to do except by using CMake; if so, then the answer is "use CMake to generate MSVC project files, don't supply your own project files".